pages/linux/du.md

```diff
@@ -1,23 +1,27 @@
 # du
 
-> Estimate file space usage.
+> Disk usage: estimate and summarize file and folder space usage.
 
-- Get a sum of the total size of a file/folder in human readable units:
+- List the sizes of a folder and any subfolders, in the given unit (B/KB/MB):
 
-`du -sh {{file_or_directory}}`
+`du -{{b|k|m}} {{path/to/folder}}`
 
-- List file sizes of a directory and any subdirectories in KB:
+- List the sizes of a folder and any subfolders, in human-readable form (i.e. auto-selecting the appropriate unit for each size):
```

Comment:
I just like to seize opportunities to expose the mechanics behind terms that are otherwise somewhat subjective :) besides, it matches the previous description to better present the case that -h should always be used unless a specific size is needed. This is important because in OS X for example the default unit is the archaic 512 bytes, which may throw people off guard -- I know it did me.